.github/workflows/ci.yml

```
@@ -0,0 +1,534 @@
+            # Extract test summary if available
+            if command -v grep >/dev/null 2>&1 && [ -f "playwright-report/index.html" ]; then
+              PASSED_TESTS=$(grep -o 'passed">[0-9]*' playwright-report/index.html | sed 's/passed">//' | head -1 || echo "0")
+              FAILED_TESTS=$(grep -o 'failed">[0-9]*' playwright-report/index.html | sed 's/failed">//' | head -1 || echo "0")
+              echo "- ✅ Passed: $PASSED_TESTS" >> $GITHUB_STEP_SUMMARY
+              echo "- ❌ Failed: $FAILED_TESTS" >> $GITHUB_STEP_SUMMARY
+            fi
+          fi
+
+      - name: 🧹 Cleanup Vercel Dev Server
+        if: always()
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:5:8: Double quote to prevent globbing and word splitting

(shellcheck)